gallery/gallery-library/src/main/java/org/silverpeas/components/gallery/MediaUtil.java

```diff
@@ -155,15 +152,14 @@ public static void processVideo(final FileHandler fileHandler, Video video,
    * @param uploadedFile the current uploaded video
    * @throws Exception
    */
-  public static void processVideo(final FileHandler fileHandler, Video video,
+  public synchronized static void processVideo(final FileHandler fileHandler, Video video,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)